src/experimental/riemannian_hmc.jl

```diff
@@ -0,0 +1,268 @@
+### integrator.jl
```

Comment:
I left a note for each code chunk on where it should if this was implemented in AHMC directly.